miio/miot_device.py

```diff
@@ -79,10 +79,11 @@ def get_properties_for_mapping(self, *, max_properties=15) -> list:
     )
     def call_action(self, name: str, params=None):
         """Call an action by a name in the mapping."""
-        if name not in self.mapping:
+        mapping = self._get_mapping()
```

Comment:
Please don't modify this file right now, but use the `status()` to access the information if necessary.

We can extend this (core) interface in the future, but that'll require some thought if it's really necessary, as well as adding unit tests for the new functionality.

Reply:
It looks like you have misplaced this comment. I haver removed `get_property` implementation and replaced usage with `self.status()` call. 

But with last changes with new `_mappings` field this change in `call_action` required. Otherwise it will try to access `self.mapping` which is not defined

Reply:
Oh, I think I commented on a single commit as clicking on the notification link brought me there, my bad...

Thanks for spotting and fixing that!